recipe/meta.yaml

```diff
@@ -36,6 +36,7 @@ outputs:
     requirements:
       build:
       host:
+        - sysroot_{{ target_platform }}
       run:
```

Comment:
We need to add the following, merge this, merge the compiler and then remove this.
```suggestion
      run:
        - sysroot_{{ target_platform }}
```

Reply:
Ahhhhhh. That's a fun game!

Reply:
I see. We need to boostrap the whole thing with the sysroot around. Makes sense.